compiler/p/codegen/BinaryEvaluator.cpp

```diff
@@ -941,7 +941,7 @@ TR::Register *OMR::Power::TreeEvaluator::imulEvaluator(TR::Node *node, TR::CodeG
 
    if (secondOp == TR::iconst || secondOp == TR::iuconst)
       {
-      int32_t value = secondChild->getInt();
+      int32_t value = secondChild->get32bitIntegralValue();
       if (value > 0 && cg->convertMultiplyToShift(node))
```

Comment:
It doesn't look like `convertMultiplyToShift` is prepared to deal with `bmul` and `smul` nodes. It only seems to have code for handling `imul` and `lmul` and will need modifications to work correctly with shorts and bytes...